pkg/distro/fedora/images.go

```diff
@@ -228,9 +220,14 @@ func osCustomizations(
 			// overwrite the profile id with the new tailoring id
 			oscapStageOptions.ProfileID = newProfile
 			oscapStageOptions.Tailoring = tailoringFilepath
+		}
 
-			// add the parent directory for the tailoring file
-			osc.Directories = append(osc.Directories, tailoringDir)
+		for _, directory := range directoriesToCreate {
```

Comment:
Maybe we can simplify this completely and just save the tailoring file to the existing `oscapDataDir`?